engine/sound/src/sound.cpp

```diff
@@ -1350,6 +1415,23 @@ namespace dmSound
         return RESULT_OK;
     }
 
+    static void SoundThread(SoundSystem* sound)
+    {
+        while (sound->m_IsRunning)
+        {
+            sound->m_Status = UpdateInternal(sound);
+            dmTime::Sleep(8000);
```

Comment:
For a 60fps game, it'll update ~twice per game frame.
We might want to tweak this value later on, or have a config setting for it.